From: Guillem Jover <gjover@sipwise.com>
Date: Tue, 1 Oct 2024 22:48:05 +0200
Subject: Do not hardcode gopsutil v4

Forwarded: Not-needed
Debian-Bug: https://bugs.debian.org/1083117

We can use v4 when no version suffix is used. Remove it so that we can
upgrade the gopsutil modules from v3 to v4 in Debian.
---
 cmd/podman/machine/init.go     | 2 +-
 cmd/podman/machine/server9p.go | 2 +-
 pkg/machine/e2e/init_test.go   | 2 +-
 pkg/machine/gvproxy_unix.go    | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/cmd/podman/machine/init.go b/cmd/podman/machine/init.go
index 4bc484f..fc2157a 100644
--- a/cmd/podman/machine/init.go
+++ b/cmd/podman/machine/init.go
@@ -13,7 +13,7 @@ import (
 	"github.com/containers/podman/v5/pkg/machine/define"
 	"github.com/containers/podman/v5/pkg/machine/shim"
 	"github.com/containers/podman/v5/pkg/machine/vmconfigs"
-	"github.com/shirou/gopsutil/v4/mem"
+	"github.com/shirou/gopsutil/mem"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"go.podman.io/common/pkg/completion"
diff --git a/cmd/podman/machine/server9p.go b/cmd/podman/machine/server9p.go
index 025145c..5095934 100644
--- a/cmd/podman/machine/server9p.go
+++ b/cmd/podman/machine/server9p.go
@@ -10,7 +10,7 @@ import (
 
 	"github.com/containers/podman/v5/cmd/podman/registry"
 	"github.com/containers/podman/v5/pkg/fileserver"
-	psutil "github.com/shirou/gopsutil/v4/process"
+	psutil "github.com/shirou/gopsutil/process"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"go.podman.io/common/pkg/completion"
diff --git a/pkg/machine/e2e/init_test.go b/pkg/machine/e2e/init_test.go
index 2ff8007..a31d892 100644
--- a/pkg/machine/e2e/init_test.go
+++ b/pkg/machine/e2e/init_test.go
@@ -14,7 +14,7 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	. "github.com/onsi/gomega/gexec"
-	"github.com/shirou/gopsutil/v4/mem"
+	"github.com/shirou/gopsutil/mem"
 	"github.com/sirupsen/logrus"
 	"go.podman.io/common/pkg/strongunits"
 )
diff --git a/pkg/machine/gvproxy_unix.go b/pkg/machine/gvproxy_unix.go
index 647572a..f5d5167 100644
--- a/pkg/machine/gvproxy_unix.go
+++ b/pkg/machine/gvproxy_unix.go
@@ -9,7 +9,7 @@ import (
 	"time"
 
 	"github.com/containers/podman/v5/pkg/machine/define"
-	psutil "github.com/shirou/gopsutil/v4/process"
+	psutil "github.com/shirou/gopsutil/process"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/sys/unix"
 )
